core/src/main/scala/cats/mtl/instances/empty.scala

```diff
@@ -328,9 +320,37 @@ trait EmptyInstances1 {
 
   implicit def traverseEmptyLiftEitherT[M[_], E](implicit under: TraverseEmpty[M]): TraverseEmpty[EitherTC[M, E]#l] = {
     new TraverseEmpty[EitherTC[M, E]#l] {
-      override val functorEmpty: FunctorEmpty[EitherTC[M, E]#l] = functorEmptyLiftEitherT(under.functorEmpty)
+
+      //implicit def functorEmptyLiftEitherT[M[_], E](implicit under: FunctorEmpty[M]): FunctorEmpty[EitherTC[M, E]#l] = {
```

Comment:
`scalastyle` wants you to put a space here before the comment body :)

Reply:
My bad, cleaned up.